test/integration/graphql/article_query_type_test.rb

```diff
@@ -25,7 +27,22 @@ class ArticleIndexTest < ActionDispatch::IntegrationTest
           edges {
             node {
               name
+              headline
+
+              author
+
               text
+              articleBody
```

Comment:
text and articleBody are the same i think, not sure if this matters

Reply:
That's fine, it's a check that articleBody also represents what we want -- it could always differ as requirements change or something :woman_shrugging: